tidb-cloud/migrate-sql-shards.md

```diff
@@ -0,0 +1,556 @@
+     port: 3307
+    ```
+
+2. Create another new data source file `dm-source2.yaml`, and add the following content:
+
+    ```yaml
+    # MySQL Configuration.
+    source-id: "mysql-replica-02"
+    # Specifies whether DM-worker pulls binlogs with GTID (Global Transaction Identifier).
+    # The prerequisite is that you have already enabled GTID in the upstream MySQL.
+    # If you have configured the upstream database service to switch master between different nodes automatically, you must enable GTID.
+    enable-gtid: true
+    from:
+     host: "192.168.10.102"
+     user: "user02"
+     password: "3O8fCPEnwO87cIal32bpO0AuTsJyBJ0="
+     port: 3308
+    ```
+
+3. Run the following command in a terminal. Use `tiup dmctl` to load the first data source configuration into the DM cluster:
+
+    ```shell
+    [root@localhost ~]# tiup dmctl --master-addr ${advertise-addr} operate-source create dm-source1.yaml
+    ```
+
+    The parameters used in the command above are described as follows:
+
+    |Parameter              |Description    |
+    |-                      |-              |
+    |`--master-addr`        |The `{advertise-addr}` of any DM-master node in the cluster where `dmctl` is to be connected. For example: 192.168.11.110:9261|
+    |`operate-source create`|Loads the data source to the DM cluster.|
+
+    The following is an example output:
+
+    ```shell
+    tiup is checking updates for component dmctl ...
+
+    Starting component `dmctl`: /root/.tiup/components/dmctl/v6.0.0/dmctl/dmctl /root/.tiup/components/dmctl/v6.0.0/dmctl/dmctl --master-addr 192.168.11.110:9261 operate-source create dm-source1.yaml
+
+    {
+       "result": true,
+       "msg": "",
+       "sources": [
+           {
+               "result": true,
+               "msg": "",
+               "source": "mysql-replica-01",
+               "worker": "dm-192.168.11.111-9262"
+           }
+       ]
+    }
+
+    ```
+
+4. Run the following command in a terminal. Use `tiup dmctl` to load the second data source configuration into the DM cluster:
+
+    ```shell
+    [root@localhost ~]# tiup dmctl --master-addr 192.168.11.110:9261 operate-source create dm-source2.yaml
+    ```
+
+    The following is an example output:
+
+    ```shell
+    tiup is checking updates for component dmctl ...
+
+    Starting component `dmctl`: /root/.tiup/components/dmctl/v6.0.0/dmctl/dmctl /root/.tiup/components/dmctl/v6.0.0/dmctl/dmctl --master-addr 192.168.11.110:9261 operate-source create dm-source2.yaml
+
+    {
+       "result": true,
+       "msg": "",
+       "sources": [
+           {
+               "result": true,
+               "msg": "",
+               "source": "mysql-replica-02",
+               "worker": "dm-192.168.11.112-9262"
+           }
+       ]
+    }
+    ```
+
+### Step 2. Create a replication task
+
+1. Create a `test-task1.yaml` file for the replication task. Configure the incremental replication mode and the starting point of the data source in the file.
```

Comment:
```suggestion
1. Create a `test-task1.yaml` file for the replication task. 
```

Reply:
"Configure the incremental replication mode and the starting point of the data source in the file." is covered in "4. Edit the task configuration file `test-task1`, to configure the incremental replication mode and replication starting point for each data source."